web/src/main.js

```diff
@@ -10,12 +10,16 @@ import store from './store';
 
 Vue.use(Girder);
 
-if (process.env.VUE_APP_SENTRY_DSN) {
-  Sentry.init({
-    dsn: process.env.VUE_APP_SENTRY_DSN,
-    integrations: [new Integrations.Vue({ Vue, logErrors: true })],
-  });
-}
+Sentry.init({
+  dsn: process.env.VUE_APP_SENTRY_DSN,
+  integrations: [
+    new Integrations.Vue({ 
+      Vue, 
+      attachProps: true,
```

Comment:
FWIW, I think this defaults to `true` if not provided, but I could see wanting it to be explicit.

Reply:
Yup, you're right, I misread the docs. I can remove, to keep it minimal.